var/spack/repos/builtin/packages/meme/package.py

```diff
@@ -35,9 +35,20 @@ class Meme(AutotoolsPackage):
 
     version('4.11.4', '371f513f82fa0888205748e333003897')
 
+    variant('mpi', default=False, description='Enable MPI support')
+
     depends_on('zlib', type=('link'))
     depends_on('libxml2', type=('link'))
     depends_on('libxslt', type=('link'))
     depends_on('libgcrypt', type=('link'))
     depends_on('perl', type=('build', 'run'))
     depends_on('python@2.7:', type=('build', 'run'))
+    # openmpi support
+    depends_on('openmpi@2:', when='+mpi', type=('link'))
+
+    def configure_args(self):
+        spec = self.spec
+        args = []
+        if '+mpi' in spec:
+            args += ['-enable-mpi']
```

Comment:
Are you sure this is the right flag? `configure --help` doesn't mention it. The only MPI related flag I found was:
```
  --with-mpidir=MPIDIR    Set MPI installation path. Default is path of found mpicc.
```